src/irmin-pack/unix/s.ml

```diff
@@ -244,6 +244,24 @@ module type S = sig
       (** [close snapshot] close the [snaphot] instance.*)
     end
   end
+
+  (** {1 Internals} *)
```

Comment:
I'm not sure how others feel, but I'm inclined to go further and wrap this part of the signature in an explicit `Internal` (or `Private`) module with the appropriate warning documentation like we do in https://github.com/mirage/irmin/blob/ae6b4fee5767b4c5e3c5c711f68e5b45281a740a/src/irmin-pack/inode_intf.ml#L125-L127